samples/bot-all-cards/csharp/BotAllCards/Resources/o365ConnectorCard.json

```diff
@@ -0,0 +1,55 @@
+﻿{
+  "@type": "MessageCard",
+  "@context": "http://schema.org/extensions",
+  "summary": "Sai Sai commented on Trello",
+  "title": "Project Tango",
+  "sections": [
+    {
+      "activityTitle": "Sai Sai commented",
+      "activitySubtitle": "On Project Tango",
+      "activityText": "\"Here are the designs\"",
+      "activityImage": "https://raw.githubusercontent.com/microsoft/botframework-sdk/master/icon.png"
+    },
+    {
+      "title": "Details",
+      "facts": [
+        {
+          "name": "Labels",
+          "value": "Designs, redlines"
+        },
+        {
+          "name": "Due date",
+          "value": "May 28, 2023"
+        },
+        {
+          "name": "Attachments",
+          "value": "[icon.png](https://raw.githubusercontent.com/microsoft/botframework-sdk/master/icon.png)"
+        }
+      ]
+    },
+    {
+      "title": "Images",
+      "images": [
+        {
+          "image": "http://connectorsdemo.azurewebsites.net/images/MicrosoftSurface_024_Cafe_OH-06315_VS_R1c.jpg"
+        },
+        {
+          "image": "http://connectorsdemo.azurewebsites.net/images/WIN12_Scene_01.jpg"
+        },
+        {
+          "image": "http://connectorsdemo.azurewebsites.net/images/WIN12_Anthony_02.jpg"
```

Comment:
Are these images free?

Reply:
I have to refer to the json and image microsoft learning only.